DEV.md

```diff
@@ -0,0 +1,41 @@
+# Dev environment
+
+
+Clone the repo and install dependencies:
+```
+$ git clone git@github.com:GaloyMoney/galoy.git
+$ cd galoy
+$ yarn install
+```
+
+We use [direnv](https://direnv.net) to load environment variables needed for running the integration tests.
+```
+$ direnv allow
+direnv reload
+direnv: direnv: loading ~/projects/GaloyMoney/galoy/.envrc
```

Comment:
not clear if this a command that need to be type or an output here

Reply:
I guess an output but `direnv allow` return nothing on my end

Reply:
its output. Lines starting with `$ ...` are input lines.

Reply:
> I guess an output but direnv allow return nothing on my end

Thats strange... I have:
```
$ direnv --version
2.28.0
```

Reply:
same version